### PR TITLE
Fix committee post close #16

### LINKED ISSUE #16: Committee post should be single year
instead of individual posts for each member, one post for the year with metabox fields for members

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

File: lib/meta-boxes.php [hunk 1]
```diff
@@ -300,5 +300,44 @@ function igv_cmb_metaboxes() {
...
+    'name'        => __( 'Name', 'cmb2' ),
+    'description' => __( '', 'cmb2' ),
+    'id'          => 'name',
+    'type'        => 'text_medium',
+  ) );
+
+  $committee_metabox->add_group_field( $committee_members, array(
+    'name'    => __( 'Bio', 'cmb2' ),
+    'id'      => 'bio',
+    'type'    => 'wysiwyg',
```
Comment (varffs) [inline lib/meta-boxes.php:334]: This has the repeating group wysiwyg issue that is still unfixed in CMB. Does this need to be wysiwyg? If not thats the simple fix.

**Comment by varffs** [comment]
Seems fine other than the issue with CMB repeating groups

**Comment by mikeyrayvon** [comment]
shit. yea it does cause of links in the bio. as in: [http://material-fair.com/en/selection-committee](url)

**Comment by mikeyrayvon** [comment]
we can certainly just say 

> fuck this

and leave them as individual posts per member, with Fair Year term

this was bretts idea.  said it would "easier" this way

**Comment by varffs** [comment]
It would be easier if not for the CMB bug :/

**Pull request closed without merging**